o.js

```diff
@@ -47,12 +46,15 @@ o.prototype.setAttribute = function (name, val){
     return this;
 }
 
+o.prototype.setAttributes = function (attributes) {
+    return this.attr(attributes);
+}
+
 o.prototype.attr = function(attrs) {
     if (Array.isArray(attrs)) {
         attrs.forEach(attr => this.element.setAttribute(attr.name, attr.val));
     } else {
         Object.entries(attrs).forEach(attr => this.element.setAttribute(attr[0], attr[1]));
```

Comment:
Wiem, że tego nie pisałeś, ale możemy to zmienić:
```suggestion
        Object.entries(attrs).forEach(([name,val]) => this.element.setAttribute(name, val));
```
Ale to też tak bardziej stylistycznie. name i val, żeby było analogicznie. Chyba, że val możnaby na value zmienić może, jak myślisz?

Reply:
Dobry pomysł